core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -597,6 +602,18 @@ private void setLocationOnce(String val, String qual, LocationType lt, boolean s
     location = new Location(val, qual, lt);
   }
 
+  private void setOperationIdOnce(String val, boolean suppressError) {
+    if (location != null || operationId != null) {
```

Comment:
Is it still an error condition if an operationId is already set and location is null?

Reply:
The way the operation id is stored in the metadata table, under normal circumstances we would never expect to see it twice for a tablet.  When we set the operation id it expected that the location and operation id are null, anything else is unexpected.